src/crepe/api/ParticleEmitter.cpp

```diff
@@ -1,36 +1,18 @@
-#include <ctime>
-#include <iostream>
-
-#include "Particle.h"
 #include "ParticleEmitter.h"
+#include "Particle.h"
```

Comment:
Je included hier Particle.h, terwijl je diezelfde file ook al include in ParticleEmiter.cpp. Daarom is de Particle.h include hier overbodig.